.github/workflows/ci-pytorch-tests.yml

```diff
@@ -184,6 +188,8 @@ jobs:
 
     - name: Testing PyTorch
       working-directory: tests/tests_pytorch
+      env:
+        PL_RUN_SLOW_TESTS: $(python -c "print(int(${{ matrix.slow }} == 'long'))")
```

Comment:
So, I'm nitpicking again, but when we have the category `quick`/`long` in the CI, I think the env variable should reflect that too (i.e. `PL_RUN_LONG_TESTS`). `slow` and `long` have too very different meanings and I'm very much not a fan of the fact, that we are introducing such a discrepancy in here.